index.html

```diff
@@ -177,5 +177,71 @@ <h3 class="title2">Skills</h3>
         </section>
       </section>
     </section>
+    <section class="form">
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+      <form id="form1" action="https://formspree.io/f/mbjwwpbw" method="post">
+        <ul>
+          <li>
+            <input
+              class="forte"
+              id="name"
+              type="text"
+              placeholder="Full name"
+              maxlength="30"
+              required
```

Comment:
Please ensure that you stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/171214785-0a750a21-d0fb-490d-b4f9-88d523871d9f.png) | ![image](https://user-images.githubusercontent.com/10332499/171214701-a04bfa95-0ed8-4979-a7e9-a59a16dae129.png)

Reply:
@micheaol Thank you for reviewing my project, I will make the necessary corrections as suggested